nmrfx-analyst/src/main/java/org/nmrfx/analyst/dataops/Align.java

```diff
@@ -439,35 +438,21 @@ public void segmentedAlign(Vec targetVec, Vec sampleVec, Vec resultVec, int preE
             targetVec.copy(subTarget, region.min, size);
             var maxIndex = subTarget.maxIndex();
             var ratio = maxIndex.getValue() / sdev;
-            int preExtra = preExtraIn;
-            int postExtra = postExtraIn;
             int maxShiftA = maxShift + Math.max(preExtraIn, postExtraIn);
-            for (int i = 0; i < nTries; i++) {
-                pt1e = region.min - preExtra;
-                pt2e = region.max + postExtra;
-                pt1e = Math.max(0, pt1e);
-                pt2e = Math.min(pt2e, vecSize - 1);
-                int sizeE = pt2e - pt1e + 1;
-                subSample.resize(sizeE);
-                sampleVec.copy(subSample, pt1e, sizeE);
-                var posValue = VecCorrelation.fftCorr(subTarget, subSample, 0, maxShiftA);
-                corrValue = posValue.getValue() / (size - 1.0);
-                shift = posValue.getPosition();
-                break;
-//                if (shift < 0) {
-//                    preExtra = preExtra - shift + 4;
-//                } else {
-//                    break;
-//                }
-//                maxShiftA = maxShift - preExtra;
-//                if (maxShiftA <  0) {
-//                    break;
-//                }
-            }
-//            if (shift < 0) {
-//                shift = 0;
-//            }
-            Alignment alignment = new Alignment(region.min, region.max, pt1e + shift, pt1e + shift + size - 1, shift, shift - preExtra, corrValue, ratio, useSection);
+            // Note: originally this method tried three times to calculate the shift to get a shift that was greater
+            // than 0 and then used a shift of 0 if the shift was still <0 but the code was commented out so removed it
```

Comment:
Removed commented out code, but left note to explain that there used to be some commented out code/for looop